### PR TITLE
Incorporate running --check into running --start

### LINKED ISSUE #348: `/scripts/elevate-cpanel --start` should run `/scripts/elevate-cpanel --check` first
- **`/scripts/elevate-cpanel --start` should run `/scripts/elevate-cpanel --check` first**

- Currently `/scripts/elevate-cpanel --start` initiates a `--check`, but, exits at the first Blocker found

- Preferred behavior:

`/scripts/elevate-cpanel --start` performs the following:

- Run `/scripts/elevate-cpanel --check`
- Create or update `/var/cpanel/elevate-blockers`
- Display the contents of `/var/cpanel/elevate-blockers` (like --check currently does)
- Exit if there are any unresolved Blockers

### PR DESCRIPTION
**Pull request by timmullin**
Fixes #348. When running start, first perform a full check for blockers, display them all to the user,
and do not start the migration if any are found.

By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

**Comment by cPholloway** [comment]
@timmullin - Can you add a changelog entry for this please?

I am going to go ahead and mark this approved pending the changelog entry.

**Review by cPholloway — APPROVED** [review]

**Comment by toddr** [comment]
oof missed the comment. Merged earlier :(

**Comment by cPholloway** [comment]
It was good to go.  The changelog was in place before the merge

**Pull request merged**
